assets/js/membraneWebRTC.ts

```diff
@@ -158,6 +158,11 @@ export interface Callbacks {
    * @param {TrackEncoding} encoding - new encoding
    */
   onTrackEncodingChanged?: (peerId: string, trackId: string, encoding: TrackEncoding) => void;
+
+  /**
+   * Called every time a local peer is removed by the server.
+   */
+  onRemoved?: (metadata: any) => void;
```

Comment:
`remove_peer` on the backend side accepts only String.t() or `nil` so we should adjust one of the sides. Either change this to `reason: string` or fix `remove_peer` on the backend to work similarly to `deny_peer`.